src/pages/transaction-signing/components/event-card.tsx

```diff
@@ -1,34 +1,35 @@
 import React from 'react';
-import { Box, color, IconButton, Stack, Text } from '@stacks/ui';
-import { SpaceBetween } from '@components/space-between';
-import { FiMoreHorizontal as IconDots } from 'react-icons/fi';
-import { Caption } from '@components/typography';
-import { AssetItem } from '@pages/transaction-signing/components/asset-item';
+import {Box, color, IconButton, Stack, Text} from '@stacks/ui';
+import {SpaceBetween} from '@components/space-between';
+import {FiMoreHorizontal as IconDots} from 'react-icons/fi';
+import {Caption} from '@components/typography';
+import {AssetItem} from '@pages/transaction-signing/components/asset-item';
+import {SendFormSelectors} from "@tests/page-objects/send-form.selectors";
 
 export const TransactionEventCard: React.FC<any> = ({
-  title,
-  left,
-  right,
-  amount,
-  ticker,
-  icon,
-  message,
-  actions,
-  isLast,
-}) => {
+                                                      title,
+                                                      left,
+                                                      right,
+                                                      amount,
+                                                      ticker,
+                                                      icon,
+                                                      message,
+                                                      actions,
+                                                      isLast,
+                                                    }) => {
```

Comment:
Make sure you're using the prettier plugin for your IDE. This is illegal formatting. I have it set up so that when I save,  it formats automatically.

Reply:
Yeah I thought I had Webstorm IDE setup to run prettier with a keyboard short cut. It reformatted but didn't include prettier I can fix this.